ci/jenkins/jobs.yaml

```diff
@@ -3,10 +3,9 @@
     repo-name: skuba
     repo-owner: SUSE
     repo-credentials: github-token
-    platform: openstack
     jobs:
         - '{name}/test'
-        - '{name}/test-vmware'
```

Comment:
I thought we were adding the libvirt, not replacing the vmware (at this stage)

Reply:
The `test` now runs on vmware (previous openstack one) and the `test-vmware` was disabled and now reenabled again but renamed. In fact I did not reenable, just deleted and created a new file, but the `git` after delete and add understood it as a rename :P